R/convert_docx.R

```diff
@@ -0,0 +1,74 @@
+#' @title Convert a docx-file (a protocol) to an (R)markdown file
+#'
+#' @description This function is derived from the \link[redoc]{dedoc} and uses pandoc to convert between docx and markdown. Several options are preset to end-up with a markdown document that is in syntax as close as possible to Rmarkdown files in RStudio. During conversion, graphics (e.g. png, jpg) will be extracted from the docx archive and placed in a folder `./media` and named `image1`, `image2`, etcetera.
```

Comment:
- Keep lines shorter than 80 characters.
- Start each sentence on a new line.

Both make any futher diff more reable. This line is truncated in the current diff...

Reply:
Resolved in 
a924080